pepper-apis/src/test/java/org/broadinstitute/ddp/route/EventServiceTest.java

```diff
@@ -395,6 +472,65 @@ private HttpResponse createTestPatchAnswerPayloadAndExecuteRequest() throws IOEx
         return response;
     }
 
+    @Test
+    public void testCopyAnswerToProfileLastName() {
+        TransactionWrapper.useTxn(handle -> {
+            JdbiProfile profileDao = handle.attach(JdbiProfile.class);
+            UserProfileDto originalProfile = profileDao.getUserProfileByUserId(testData.getUserId());
+
+            AnswerDao answerDao = AnswerDao.fromSqlConfig(ConfigFactory.parseResources(ConfigFile.SQL_CONF));
+            String lastNameFromAnswer = "Sargent" + Instant.now().toEpochMilli();
+            Answer expected = new TextAnswer(null, copySourceStableId, null, lastNameFromAnswer);
+            answerDao.createAnswer(handle, expected, testData.getUserGuid(), copyInstanceGuid);
+
+            //Let's make sure they start being different
+            assertNotEquals(originalProfile.getLastName(), expected.getValue());
+
+            EventService.getInstance().processSynchronousActionsForEventSignal(
+                    handle,
+                    new ActivityInstanceStatusChangeSignal(testData.getUserId(),
+                            testData.getUserId(),
+                            testData.getUserGuid(),
+                            copyInstanceId,
+                            copyActivityId,
+                            testData.getStudyId(),
+                            InstanceStatusType.COMPLETE));
+
+            UserProfileDto profile = profileDao.getUserProfileByUserId(testData.getUserId());
+
+            assertEquals(lastNameFromAnswer, profile.getLastName());
```

Comment:
Looks like this `testCopyAnswerToProfileLastName` test is causing `GetDsmStudyParticipantTest` tests to fail since we changed the last name here. I think the simplest thing to do is add
```
handle.rollback();
```
right after this `assertEquals` so we don't commit the changes in this transaction.